doc/rose-rug-advanced-tutorials-bunch.html

```diff
@@ -0,0 +1,341 @@
+    </div>
+
+    <div id="currentSlide">
+      <!-- DO NOT EDIT -->
+    </div>
+
+    <div id="controls">
+      <!-- DO NOT EDIT -->
+    </div>
+
+    <div class="header-footer" id="footer"></div>
+  </div>
+
+  <div id="body-main" class="presentation">
+    <div class="slide">
+      <h1>Rose Advanced Tutorial: Rose Bunch</h1>
+    </div>
+
+    <div class="handout" id="content"></div>
+
+    <div class="slide">
+      <h2 id="intro">Introduction</h2>
+
+      <p>This part of the Rose user guide walks you through using the <a href=
+      "rose-rug-task-run.html#rose-task-run.util.rose_bunch">Rose Bunch</a>
+      built in app.</p>
+
+      <p>This allows multiple variants of a command to be run under a single
+      job.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="purpose">Purpose</h2>
+
+      <p>Often, we want to run many instances of a command that differ only
+      slightly from each other at the same time - an example would be where a
+      command is run repeatedly with only its arguments changing.</p>
+
+      <p>Rather than creating multiple apps or optional configs to change the
+      way a command is to be run, we can instead use the built-in <var>rose
+      bunch</var> application to run multiple command variants, in parallel,
+      under a single job as defined by an application configuration.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="purpose">Warning</h2>
+
+      <p>It is important to note that when running your <var>rose bunch</var>
+      app under load balancing systems such as PBS or Slurm, you will need to
+      set resource requests to reflect the resources required by running
+      multiple commands at once.</p>
+
+      <p>For example, if a single command would require 1GB memory and the app
+      is configured to run up to 4 commands at once then 4GB of memory should
+      be requested.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="example-task">Example</h2>
+
+      <p>In this example we are going to create a suite that simulates the
+      handling of landing planes at an airport. For a given plane the process
+      of landing and unloading is the same: land, taxi to the terminal, unload
+      passengers and get clear. We can refer to this as the "landing" routine.
+      What differs between landings is the plane type, number of passengers
+      carried and the resulting timings for each stage of the landing
+      process.</p>
+
+      <p>Create a <a href="rose-rug-brief-tour#cli">new suite</a> (or just a
+      new directory somewhere - e.g. in your homespace) containing a blank
+      <samp>rose-suite.conf</samp> and a <samp>suite.rc</samp> file that looks
+      like this:</p>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example suite.rc</h3>
+      <pre class="prettyprint lang-cylc">
+[cylc]
+    UTC mode = True # Ignore DST
+    [[event hooks]]
+        timeout handler = rose suite-hook --mail --shutdown
+        timeout = P3D
```

Comment:
You may want to remove these event hooks. They don't add value to the tutorial, and we can now rely on individual sites to have these configured to suit their needs.